sematic/ui/src/pipelines/NotesPanel.tsx

```diff
@@ -135,7 +135,7 @@ export default function NotesPanel() {
                 <NoteView
                   note={note}
                   key={idx}
-                  author={authorsByEmailRef.current.get(note.author_id) || anonymousUser}
+                  author={authorsByUserIdRef.current.get(note.user_id) || anonymousUser}
```

Comment:
I don't believe this is going to work. If you look at
https://github.com/sematic-ai/sematic/blob/be96c352fad310858a8bd598df31ead4989af0b7/sematic/api/endpoints/notes.py#L53-L59
You can see the `authors` field will be removed after this UI chance.

Instead, you want to change the `Note` model in `Models.tsx` to have a `user: User | undefined` field (see `Run`), and use that instead.

Reply:
Got it! Fixing now.

Reply:
How does it look now?